src/socialMediaActivity.js

```diff
@@ -931,10 +1143,19 @@ function verifyFacebookReshare({requestDetails = null }) {
     return null;
 }
 
+/**
+ * An object representing the contents of a post as returned by Facebook.
+ * @typedef facebookPostContentsDetails
+ * @param {string} postText - the plain text of the post.
+ * @param {string[]} postAttachments - any links attached to the post.
+ */
+
 /**
  * Get the contents and attachments of a Facebook post.
- * @param postId - the unique ID of the post
- * @param ownerId - the unique ID of the owner, or of the group, if the post is in a group
+ * Note: Do not pass Facebook post IDs as normal numbers, as they are too large to
+ *  maintain precision. Use strings instead.
+ * @param {string} postId - the unique ID of the post
+ * @returns {facebookPostContentsDetails}
  */
 export function getFacebookPostContents(postId) {
     return new Promise((resolve, reject) => {
```

Comment:
Same comment as the similar functionality for Twitter. This seems like it could be handled in the background page.